doc/use_cases/turtlebot3_tutorial.md

```diff
@@ -0,0 +1,239 @@
+
+### 1- Launch Turtlebot 3 Robot State Publisher
+
+Launch Turtlebot 3 robot state publisher,
+
+    ros2 launch turtlebot3_bringup turtlebot3_state_publisher.launch.py use_sim_time:=True
+
+Make sure use_sim_time is set to **False**.
+
+### 2- Launch Navigation2 
+ 
+ You need to have a map of the environment where you want to Navigate Turtlebot 3. Required files:
+
+   - ```your-map.map```
+   - ```your-map.yaml```
+
+Launch Navigation 2. If you set autostart:=False, you need to click on the start button in RVIZ to initialize the nodes. Make sure `use_sim time` is set to **False**, because we want to use the system time instead of the time simulation time in Gazebo.
+
+`<your_map>.yaml` is the configuration file for the map we want to provide Navigation2. In this case, it has the map resolution value, threshold values for obstacles and free spaces, and a map file location. You need to make sure these values are correct. More information about the map.yaml can be found [here](http://wiki.ros.org/map_server).
+
+```
+ros2 launch nav2_bringup nav2_bringup_launch.py use_sim_time:=False autostart:=False \
+    map:=/home/<user_name>/navigation2_ws/install/nav2_bringup/share/nav2_bringup/launch/`your_map`.yaml
+```
+Again don't forget to change **<user_name>** to your actual user name. 
+
+### 3-  Launch RVIZ
+
+Launch RVIZ with a pre-defined configuration file.
+
+    ros2 run rviz2 rviz2 -d $(ros2 pkg prefix nav2_bringup)/share/nav2_bringup/launch/nav2_default_view.rviz
+
+Now, you should see a shadow of Turtlebot 3 robot model in the center of the plot in Rviz. Click on the Start button (Bottom Left) if you set the auto_start parameter to false.
+
+![enter image description here](https://github.com/mlherd/ros2/blob/master/img/Rviz_start.PNG?raw=true)
+
+Now, the map should appear in Rviz.
+
+![enter image description here](https://github.com/mlherd/ros2/blob/master/img/rviz_custom_map.png?raw=true)
+
+
+### 4- Initialize the Location of Turtlebot 3
+
+First, find where the robot is on the map. Check where your robot is in the room.
+
+Set the pose of the robot in Rviz. Click on the 2D Pose Estimate button and point the location of the robot on the map. The direction of the green arrow is the orientation of Turtlebot.
+  
+![enter image description here](https://github.com/mlherd/ros2/blob/master/img/rviz_2d_pose_extimate_custom_map.png?raw=true)
+
+Now, the 3D model of Turtlebot should move to that location. A small error in the estimated location is tolerable.
+
+### 5-  Send a Goal Pose
+
+Pick a target location for Turtlebot on the map. You can send Turtlebot 3 a goal position and a goal orientation by using the **Navigation2 Goal** the **GoalTool** buttons.
+
+*Note*: Difference between buttons. Navigation2 Goal button uses a ROS2 Action to send the goal and GoalTool publishes the goal to a topic.
```

Comment:
Theres alot of copy pasted stuff, I think you should number the steps and just tell them to reference N through N+M after bringup from simulation

Reply:
Actually, that was my initial design for this tutorial, but in my opinion reading this type of tutorials is usually easier for the user, so they don't have to jump to any steps.

I agree that there is some avoidable duplication especially  the last two steps. I think I will end up splitting this tutorial into two separate tutorials.

Reply:
I did the separation, but there is still copy/paste stuff in the tutorials. I think the number of duplications in the documentation will decrease as we slowly move these tutorials to Github pages.